foreman_wreckingball.gemspec

```diff
@@ -13,11 +13,13 @@ Gem::Specification.new do |s|
   # also update locale/gemspec.rb
   s.description = 'Adds status checks of the VMWare VMs to Foreman.'
 
+  s.required_ruby_version = '>= 2.5', '< 4'
```

Comment:
We haven't really supported 2.5 for a long time. Would it e better to move to 2.7+

Reply:
this wait's for https://github.com/theforeman/theforeman-rubocop/pull/8/files